lib/validators/specResolver.ts

```diff
@@ -211,7 +210,7 @@ export class SpecResolver {
       this.specInJson = resolveNestedDefinitions(this.specInJson, this.options)
 
       // all transformations without dependencies should be moved here)
```

Comment:
redundant comment ?

Reply:
if there's was a question about the comment, maybe the comment should be clarified?